src/Proxies/WP.php

```diff
@@ -166,4 +166,15 @@ public function wp_timezone(): DateTimeZone {
 	public function number_format_i18n( float $number, int $decimals = 0 ): string {
 		return number_format_i18n( $number, $decimals );
 	}
+
+	/**
+	 * Determines whether the current request is a WordPress Ajax request.
+	 *
+	 * @return bool True if it's a WordPress Ajax request, false otherwise.
+	 *
+	 * @since x.x.x
```

Comment:
❓ 

x.x.x ? I guess to be replaced by 2.0.0?

Reply:
Yeah, that's probably right!
```suggestion
	 * @since 2.0.0
```